docs/project/faq.md

```diff
@@ -486,36 +488,59 @@ other parts of LLVM will be advantageous, but not required.
+community should treat it as _firmly_ decided. This doesn't mean that the
+decision is _definitely_ right or set in stone, it just means we'd like the
+community to focus time and energy on other issues in the name of progress.
 
 Sometimes, it will be appropriate to revisit a decision; for example, when there
 is new information introduced, a new community joins Carbon, or there is an
-order of magnitude growth in the community. We will do this by filing an issue
-for leads with what new information is available, and we will reevaluate the
-choice.
+order of magnitude growth in the community. These cases are handled as new
+proposals through the [evolution process](evolution.md).
 
-For example, we're doing this with digit separators: we missed important
+For example, we have done this with digit separators: we missed important
 _domain_ conventions and had overly restricted where separators are allowed,
 [an issue was filed with the new information](https://github.com/carbon-language/carbon-lang/issues/1485),
 and we're fixing the choice.
 
-When there is a discussion about options or disagreement with a decision,
-comments saying "I like X" and "I don't like X" do not contribute to the
-conversation. You should use the reactions to signal this kind of sentiment, and
-we'll pay attention to those. Beyond that, we encourage folks to either shrug
-and let quirky syntax be quirky, or look for the objective use case that was
-clearly missed in the previous decision and makes the current state _sharply_
-problematic before pushing for change.
+See also the related questions
+[What happens when a decision was wrong?](#what-happens-when-a-decision-was-wrong),
+[How does Carbon make decisions?](#how-does-carbon-make-decisions) and
+[What can I do if I disagree with a design decision?](#what-can-i-do-if-i-disagree-with-a-design-decision).
+
+### What can I do if I disagree with a design decision?
+
+We invite you to give us constructive feedback. Some of Carbon's design
+decisions are made with the _expectation_ of receiving community feedback. We
+understand that many decisions won't be universally popular, but we'd still like
+to understand the community's reaction to Carbon.
+
+We encourage you to investigate why Carbon came to be the way it is. Look for
+proposals, open issues, and past discussions. Read through and understand the
+context and rationale behind the decisions you are concerned about. You may find
+that your concerns were already thoroughly discussed. If not, you will be in a
+better place to present your thoughts in a convincing way.
+
+The Carbon language is still young, and there are aspects of the language that
+are provisional and expected to change.
+
+Changing decisions that have come out of the [evolution process](evolution.md)
+involves a formal process. See
+[When do we re-visit decisions or re-open discussions?](#when-do-we-re-visit-decisions-or-re-open-discussions).
```

Comment:
```suggestion
[When do we revisit decisions or reopen discussions?](#when-do-we-revisit-decisions-or-reopen-discussions).
```

Because my other suggestion will break these